docs/modules/ROOT/pages/configuration/delayed-deletes.adoc

```diff
@@ -0,0 +1,140 @@
+= Delayed Deletes
+
+This document describes how to configure KubeArchive when resources need
+to be deleted after a certain time.
+
+KubeArchive features vacuums that reescan namespaces for stale or missed
+resources. Vacuums in combination with filters that include time expressions
+allow users to delay the delete of resources. More information about Vacuums
+xref:/reference/vacuum.adoc[here].
+
+This feature is useful for resources that are monitored or modified by other
+systems without any flag on the resource itself that can be used by KubeArchive.
+For example metadata collection systems, build signature systems...
```

Comment:
```suggestion
For example metadata collection systems, build signature systems, etc.
```

Reply:
We don't like latin abbreviations: https://kubearchive.github.io/kubearchive/main/contributors/documentation.html#_latin_abbreviations